### DISCUSSION TITLE
AMD hardware question?

### DISCUSSION THREAD

**Discussion by Welsyntoffie** [blakeblackshear/frigate#7855] [Ask A Question]
I have an AMD cpu laying around. fX series (fd4170FRW4KGU). I read that this cpu has no build in graphics, but does support AVX.
I also have ATi gpu HD4550. 
I'm an intel person, I have no clue about amd. I assume the gpu will be supported as its better than HD2000 minimum requires as per docs. 

So my questions comes down to...
Will this hardware combination work with frigate? Do i need the GPU with this cpu or could I use the cpu alone.

I currently have frigate running on an i3 6th gen cpu with hardware accel on at 88ms inference.

I get to busy with work to try and test the amd setup, which is why I'm asking here.

**Comment by NickM-27** [comment]
> Will this hardware combination work with frigate? Do i need the GPU with this cpu or could I use the cpu alone.

I think it *should* work but you for one definitely need the GPU. Secondly, you'll need a coral to do the object detection because AMD accelerated object detection isn't supported.

**Reply by Welsyntoffie** [reply]
Thanks for the info. A coral is still in my dreams. USB ones are priced high. Im trying to get a mini pcie one. 
So for now I'll just keep using the i3 laptop. Its doing n decent job with 3 cameras.
